automation_tools/__init__.py

```diff
@@ -2156,10 +2156,16 @@ def product_install(distribution, certificate_url=None, selinux_mode=None, sat_v
 
     # enable cockpit feature
     if version(sat_version) > version(6.6):
-        # WORKAROUND https://bugzilla.redhat.com/show_bug.cgi?id=1889320 manifests only on 6.8
-        if not bz_bug_is_open(1889320) or sat_version != '6.8':
+        # WORKAROUND https://bugzilla.redhat.com/show_bug.cgi?id=1889320 manifests only on 6.8+
+        if not bz_bug_is_open(1889320) or sat_version not in ['6.8', '6.9']:
             installer_options.update({'enable-foreman-plugin-remote-execution-cockpit': None})
 
+    # WORKAROUND for 6.9 presnaps
+    if sat_version == '6.9':
+        execute(lambda: run(
```

Comment:
I think upcoming snap won't be a presnap but an actual snap, so I don't think this is still needed as per comment on 6.9-PresSnapIssues sheet.